web/js/pages/experiment/common/api/page.js

```diff
@@ -46,11 +45,6 @@ const CORE_MODULE_CODE_MAP = {
       <PythonExperimentCodeGen {...props} />
     </CopyableText>
   ),
```

Comment:
It will be a little awkward having only 1 option in the language selection but 🤷